src/components/training/TrainingPlanOverview.jsx

```
@@ -0,0 +1,1114 @@
+                        ? "border-blue-500/50 bg-blue-500/10 ring-2 ring-blue-500/20"
+                        : "border-white/10 hover:border-white/20"
+                    }`}
+                    onClick={(e) => {
+                      if (
+                        e.target === e.currentTarget ||
+                        e.target.closest(".day-main-content")
+                      ) {
+                        toggleDayExpansion(dayIdx);
+                      }
+                    }}
+                  >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)